### ISSUE TITLE
inability to paste in the API key

### ISSUE THREAD

**Issue by marrie** [xor-gate/syncthing-macosx#26]
I'm a visually impaired user who is trying this thing out. I went to my extras and tried to paste in the very long APK key that was given to me under syncthing's advanced area under the web interface.  With voiceover on I cannot paste it in. Can someone test this with voiceover? You can invoke voiceover with command plus f5 from anywhere.Once I get this working I know I'll be able to test for more issues in regard to accessibility and functionality.

Thanks.

**Comment by xor-gate** [comment]
Probably related to #20

**Comment by xor-gate** [comment]
Continue in the old thread of #20
